src/ui/common/src/components/pages/workflow/id/index.tsx

```diff
@@ -334,9 +337,12 @@ const WorkflowPage: React.FC<WorkflowPageProps> = ({
       <Box
         sx={{
           display: 'flex',
-          width: contentWidth,
+          // TODO: just create a state variable to reflect the open state of the drawer.
```

Comment:
drawer open state is just whether or not a node is selected should be easy to define a constant in this page.